src/guides/bioinformatics/exporting-data-for-auspice.md

```diff
@@ -0,0 +1,149 @@
+
+```js
+// node-data file 1
+nodes: {NODE_NAME: {country: "USA", region: "North America"}}
+// node-data file 2
+nodes: {NODE_NAME: {country: "Canada"}}
+// metadata TSV 
+{NODE_NAME: {country: "Mexico", serotype: "ABC"}}
+// resulting merged metadata
+nodes: {NODE_NAME: {country: "Canada", region: "North America", serotype: "ABC"}}
+```
+
+Note that for backwards compatibility, some keys are changed as they are read. Currently `authors → author` and  `numdate → num_date`.
+
+**What are the colorings, filters and geo-resolutions?**
+
+Dataset colorings are created via the combination of:
+* All key-value pairs specified in node-data JSONs, unless they match a hardcoded-list of exclusions. This has the effect that data provided in node-data JSONs will form a colouring and thus be exported in the dataset JSON.
+* Command-line provided colorings
+* Colorings specified in the auspice-config JSON.
+* Special cases are automagically added if the provided data support them:
+    * `gt` if mutations are defined.
+    * `num_date` if dates are defined (on any nodes)
+    * `clade_membership` if `clade_membership` or `clade_annotation` are defined on any nodes.
+	 
+Note that if titles or scale types are not provided (the auspice-config JSON is the only way to provide this information) then we maintain a small list of defaults to use, for instance `num_date` gets the title “Sampling date”. If in doubt, it’s safest to provide this information in the auspice-config JSON.
+
+Filters are defined via either the “filters” section of an auspice-config JSON, or if this is not provided/set, then all non-continuous colorings (except `gt`). 
+
+Geo-resolutions are those set via the command line or via the `geo_resolutions` block of an auspice-config JSON. 
+
+**So… which pieces of metadata are assigned to nodes?**
+
+After combining the metadata inputs and defining colorings, filtering and geo-resolutions, the final step is to actually assign valid key-value pairs [4] to each node in the dataset JSON. The following steps are used:
+
+* `muts` (nucleotide mutations, thus set as `node.branch_attrs.nuc`) and `aa_muts` (keys therein set as keys in `node.branch_attrs`) [5].
+* `vaccine`, `url` and `accession`, `hidden` are transferred over to `node.node_attrs`.
+* `clade_annotation` -> `node.branch_attrs.labels.clade`
+* `num_date`  set to `node.node_attrs.num_date`. If `num_date_confidence` present then this is associated with this data.
+* `mutation_length` or `branch_length`  is mapped to `node.node_attrs.div` [6]
+* Keys `journal`, `author`, `authors`, `title` and `paper_url` are assigned to `node_data.author: Dictionary<string>`  [7].
+* Any specified colorings, filters or geo-resolutions (see above) are transferred over (unless they are those in the previous bullet points). If there exists companion keys `${key}_confidence` or `${key}_entropy` these are associated with the original key.
```

Comment:
If this is an ordered set of steps, then I'd suggest using a numbered list to make that clear.

Reply:
It is safe to read this as an ordered set of steps, so I've changed to this 👍